4_turb_abl_fsi/template_files/static_files/nrel5mw.yaml

```diff
@@ -0,0 +1,136 @@
+# this may be excessive use of anchors but if we change
+# names and such we can do it all here
+turbine_info: 
+  turb0:
```

Comment:
we need something for the mesh tag offset - https://github.com/Exawind/exawind-cases/blob/main/1_turb_abl_fsi/template_files/nrel5mw_nalu.yaml#L183

`mesh_tag_offset: &tag_0 0` ?